orttraining/orttraining/python/training/ortmodule/torch_cpp_extensions/install.py

```diff
@@ -41,9 +41,7 @@ def _install_extension(ext_name, ext_path, cwd):
 def build_torch_cpp_extensions():
     """Builds PyTorch CPP extensions and returns metadata."""
     # Run this from within onnxruntime package folder
-    is_gpu_available = torch.cuda.is_available() and (
-        ortmodule.ONNXRUNTIME_CUDA_VERSION is not None or ortmodule.ONNXRUNTIME_ROCM_VERSION is not None
-    )
+    is_gpu_available = ortmodule.ONNXRUNTIME_CUDA_VERSION is not None or ortmodule.ONNXRUNTIME_ROCM_VERSION is not None
```

Comment:
```suggestion
    is_gpu_available = (torch.version.cuda is not None or torch.version.hip is not None) and (ortmodule.ONNXRUNTIME_CUDA_VERSION is not None or ortmodule.ONNXRUNTIME_ROCM_VERSION is not None)
```

Reply:
Let's try @baijumeswani idea to fix the issue instead of revetting. `torch/version.py` is generated during PyTorch packaging, so we don't really need hardware support during `docker build` for this check!

Reply:
If it works, please update the PR name and description